README.md

```diff
@@ -30,4 +30,6 @@ Information about adding
 * an additional MicroProfile Version for an Implementation
 * a new MicroProfile Specification
 
-is found in the [how to document](https://github.com/eclipse/microprofile-starter/blob/master/how-to.md).
\ No newline at end of file
+is found in the [how to document](https://github.com/eclipse/microprofile-starter/blob/master/how-to.md).
+
+It is also possible to build and deploy to Open Liberty using the Liberty profile. To build the MP Starter app, use: `mvn package -Pliberty liberty:run` and then browse to http://localhost:9080/mp-starter/ to test the app.
```

Comment:
You should add the above info [here](https://github.com/eclipse/microprofile-starter/tree/master/Container)

Reply:
Thanks for the suggestion @Emily-Jiang - I have moved it there.